pkg/mpc-examples/test/00-NullController.test.ts

```diff
@@ -0,0 +1,163 @@
+
+let deployer: SignerWithAddress, rando: SignerWithAddress;
+let mpcFactory: Contract;
+let tokenAddresses: string[];
+
+const initialWeights = toNormalizedWeights([fp(30), fp(70)]);
+const swapFeePercentage = bn(0.3e16);
+
+async function deployBalancerContract(
+  task: string,
+  contractName: string,
+  deployer: SignerWithAddress,
+  args: unknown[]
+): Promise<Contract> {
+  const artifact = await getBalancerContractArtifact(task, contractName);
+  const factory = new ethers.ContractFactory(artifact.abi, artifact.bytecode, deployer);
+  const contract = await factory.deploy(...args);
+  return contract;
+}
+
+async function deployController(deployer: SignerWithAddress): Promise<Contract> {
+  const newPoolParams = {
+    name: 'MyTestPool',
+    symbol: 'MTP',
+    tokens: tokenAddresses,
+    normalizedWeights: initialWeights,
+    // assetManagers: [ZERO_ADDRESS, ZERO_ADDRESS],
+    swapFeePercentage: swapFeePercentage,
+    swapEnabledOnStart: true,
+    // mustAllowlistLPs: false,
+    managementAumFeePercentage: fp(0.1),
+    aumFeeId: 0,
+  };
+
+  const receipt = await (await mpcFactory.connect(deployer).create(newPoolParams)).wait();
+  const eventController = expectEvent.inReceipt(receipt, 'ControllerCreated');
+
+  return ethers.getContractAt('NullController', eventController.args.controller);
+}
+
+/* This fn will be useful for future tests, though it is extraneous here
+async function fastForward(sec: number) {
+  const mostRecentBlock = await ethers.provider.getBlockNumber();
+  const timestamp = (await ethers.provider.getBlock(mostRecentBlock)).timestamp;
+  await ethers.provider.send('evm_mine', [timestamp + sec]);
+}
+*/
+
+async function deployBalancerManagedPoolFactory(
+  task: string,
+  libNames: string[],
+  contractName: string,
+  deployer: SignerWithAddress,
+  args: unknown[]
+): Promise<Contract> {
+  const lib_cb = await deployBalancerContract(task, libNames[0], deployer, []);
+  const lib_art = await deployBalancerContract(task, libNames[1], deployer, []);
+  const libraries = {
+    CircuitBreakerLib: lib_cb.address,
+    ManagedPoolAddRemoveTokenLib: lib_art.address,
+  };
+
+  const artifact = await getBalancerContractArtifact(task, contractName);
+  const factory = await ethers.getContractFactoryFromArtifact(artifact, { signer: deployer, libraries });
+  const instance = await factory.deploy(...args);
+  return instance.deployed();
+}
+
+async function deployLocalContract(contract: string, deployer: SignerWithAddress, args: unknown[]): Promise<Contract> {
+  const Controller = await ethers.getContractFactory(contract);
+  const c = await Controller.connect(deployer).deploy(...args);
+  const instance: Contract = await c.deployed();
+  return instance;
+}
+
+describe('NullController', function () {
+  let vault: Contract;
+  let tokens: TokenList;
+
+  before('Setup', async () => {
+    /* eslint-disable @typescript-eslint/no-unused-vars */
+    let trader;
```

Comment:
Is this lint hint necessary? The variable seems to be used. Also, let's give it a type; I think it's `SignerWithAddress`?

Reply:
This was vestigial. Fixed it now.